apps/frontend/src/libs/helpers/get-user-rows/get-user-rows.helpers.ts

```diff
@@ -0,0 +1,10 @@
+import { type TableUser } from "~/libs/types/types.js";
```

Comment:
this shouldn't be in common helpers, should be in page helpers the same for TableUser (let's rename it to UserRow)

Reply:
Maybe move the get-user-columns.helper.ts file there too?

Reply:
yes